app/controllers/notes_controller.rb

```diff
@@ -103,7 +103,11 @@ def create
         else
           flash[:notice] = "Research note published. Get the word out on <a href='/lists'>the discussion lists</a>!"
         end
-        redirect_to @node.path
+        if params[:redirect] && params[:redirect] == 'question'
```

Comment:
Oh yes we can surely do that by changing the path for questions in the `generate_path` method. But that would mean users could no longer view the as normal research notes. Thus it would affect the present case if deployed on the main website, thus affecting the backward compatibility. So I devised a way so that question could co-exist in the research notes page as well as in the questions page.